### PR TITLE
Modify Nor Flash operation for iMXRT1020

### PR DESCRIPTION
**Pull request by HarrisonChoi750405**
1. verification for flash page write

### CODE REVIEW

**Comment by seinfra** [comment]
Target : [9b47447db474d247677f71498f768b1bfa7e4e5a] - Code Rule Check (C++) OK.

**Comment by seinfra** [comment]
Target : [9b47447db474d247677f71498f768b1bfa7e4e5a] - Code Rule Check OK.

File: os/arch/arm/src/imxrt/imxrt_flash.h [hunk 1]
```diff
@@ -136,6 +136,7 @@ extern "C" {
 #define EXTERN extern
 #endif
 
+#if !defined(CONFIG_ARCH_CHIP_FAMILY_IMXRT102x)
```
Comment (chanijjani) [inline os/arch/arm/src/imxrt/imxrt_flash.h]: Why did you add this..? This function seems to be used only for imxrt1020 board...

File: os/arch/arm/src/imxrt/imxrt_norflash.c [hunk 2]
```diff
@@ -725,6 +702,7 @@ static inline void flexspi_clock_init(void)
 /************************************************************************************
  * Public Functions
  ************************************************************************************/
+#if !defined(CONFIG_ARCH_CHIP_FAMILY_IMXRT102x)
```
Comment (chanijjani) [inline os/arch/arm/src/imxrt/imxrt_norflash.c]: Why did you add this..? This function seems to be used only for imxrt1020 board...

**Comment by chanijjani** [comment]
@HarrisonChoi750405 We'd better update PR #3273 and close this.
